### PR TITLE
New zingolib & a little fix in header

### PR DESCRIPTION
**Pull request by juanky201271**

### CODE REVIEW

File: rust/Cargo.toml [hunk 1]
```diff
@@ -19,5 +19,5 @@ opt-level = 3
 debug = false
 
 [patch.crates-io]
-zcash_note_encryption = { git = "https://github.com/zingolabs/librustzcash", rev = "acfdabc3629c93517e3c4742848996c595de6d3c" }
-zcash_primitives = { git = "https://github.com/zingolabs/librustzcash", rev = "acfdabc3629c93517e3c4742848996c595de6d3c" }
+zcash_note_encryption = { git = "https://github.com/zingolabs/librustzcash", rev = "fa67d9a645da55265ebea8cedbb77ff42317c726" }
```
Comment (Oscar-Pepper) [inline rust/Cargo.toml]: Can we match the cargo toml to zingolib so dependencies are the same
https://github.com/zingolabs/zingolib/blob/dev/Cargo.toml

**Review by Oscar-Pepper — CHANGES_REQUESTED** [review]
I think we want to only depend on libraries like librustzcash from zingolib... Ideally we should remove these deps from zingo-mobile but for now let's at least make sure they match

File: rust/Cargo.toml (hunk 1 above)
Comment (juanky201271) [reply rust/Cargo.toml:22]: the commit `fa67d9a645da55265ebea8cedbb77ff42317c726` is the same in both, right?

Comment (juanky201271) [reply rust/Cargo.toml]: I removed them.

**Comment by juanky201271** [comment]
> I think we want to only depend on libraries like librustzcash from zingolib... Ideally we should remove these deps from zingo-mobile but for now let's at least make sure they match

I removed it, ready for review... @Oscar-Pepper

**Review by zancas — APPROVED** [review]

**Comment by zancas** [comment]
Nice catch @Oscar-Pepper and @juanky201271

**Review by Oscar-Pepper — APPROVED** [review]

**Pull request merged**